code/modules/tgui/plugins/login.dm

```diff
@@ -135,6 +136,10 @@ GLOBAL_LIST(tgui_logins)
 		var/mob/living/silicon/robot/R = usr
 		state.name = usr.name
 		state.rank = "[R.modtype] [R.braintype]"
+	else if(login_type == LOGIN_TYPE_ADMIN && usr.can_admin_interact())
+		state.name = "CentComm Secure Connection"
```

Comment:
A few things:
- If you set someone to arrest, what shows in their records as the person who made the change?
- This should set the access list of the person logged in to all_access + get_all_centcom_access.

Reply:
![image](https://user-images.githubusercontent.com/1496804/94130616-31415400-fe5d-11ea-9759-9f697eac1b16.png)